### PR TITLE
Update build script to upload archives to Maven

### PR DESCRIPTION
**Pull request by wangmengyan95**
1. name
`ParseUI-Login-Android` and `ParseUI-Widget-Android`

2. artifactId
`parseui-login-android` and `parseui-widget-android`

3. packaging
`aar` and `jar`

4. version
0.0.1 and 0.0.1

### CODE REVIEW

File: ParseUI-Widget/build.gradle [hunk 1]
```diff
@@ -36,5 +40,124 @@ android {
...
+        exclude '**/oauth/**'
+    }
+
+    def javadocJarTask = task("javadocJar${variant.name.capitalize()}", type: Jar, dependsOn: "javadoc${variant.name.capitalize()}") {
+        classifier = 'javadoc'
+        from javadocTask.destinationDir
+    }
+
+    if (name.equals(BuilderConstants.RELEASE)) {
+        artifacts.add('archives', jarTask);
```
Comment (grantland) [inline ParseUI-Widget/build.gradle]: We probably don't want a jar task for this, since it may include xml resources in the future.

Comment (wangmengyan95) [inline ParseUI-Widget/build.gradle]: kk, we will also package it as aar right?

Comment (grantland) [inline ParseUI-Widget/build.gradle]: yes

**Comment by grantland** [comment]
LGTM, feel free to squash and merge!

**Pull request merged**